common/constants.go

```diff
@@ -2,4 +2,5 @@ package common
 
 const (
 	InternalDataSourcePath = "pkmn.csv"
+	TestDataSourcePath     = "pkmn_test.csv"
```

Comment:
You would like to keep the test related stuff within the context of the test files. It's ok to define this as a constant, but inside the appropriate `*_test.go` file